napari/utils/colormaps/_tests/test_colormap.py

```diff
@@ -27,6 +28,38 @@ def test_linear_colormap_with_control_points():
     np.testing.assert_almost_equal(cmap.map([0.75]), [[0, 1, 0, 1]])
 
 
+def test_non_ascending_control_points():
+    """Test non ascending control points raises an error."""
```

Comment:
Is there a benefit to adding the docstring here and in other tests? To me the name of the test and its body are self-descriptive and read well, so the docstring doesn't seem to add much and I'd remove it.

I see docstrings for tests being more useful when the test is complex or when it refers to an issue (e.g. on GitHub), but I might be missing something.

Reply:
it's a very valid question - i think we have a non-uniform adoption of doc strings in our tests. some are longer, some are shorter. i agree this one doesn't help much, but i might still be inclined to leave for now as i don't see it hurting.

if someone else also weighed in to say we should drop them then i'd happily do so. i don't have strong feelings here at all though. we could do an audit of our test docs strings at some point too and standardize things/ clean them up